src/main.ts

```diff
@@ -295,6 +315,24 @@ function handleKeyDown({ key }: KeyboardEvent): void {
   }
 }
 
+async function handleFindTrack() {
+  // Load track value
+  let trackId = trackInput.valueAsNumber;
+  let newTrack = dataset!.buildTrack(trackId);
```

Comment:
Please always prefer `const` over `let` since it is more conservative against accidental modification.  (We probably should add this to our linting rules if not already)

Reply:
Fixed and left a note on the issue ticket about adding linting for const/let